client/trino-client/src/main/java/io/trino/client/uri/ConnectionProperties.java

```diff
@@ -97,6 +97,7 @@ enum SslVerificationMode
     public static final ConnectionProperty<String, Class<? extends DnsResolver>> DNS_RESOLVER = new Resolver();
     public static final ConnectionProperty<String, String> DNS_RESOLVER_CONTEXT = new ResolverContext();
     public static final ConnectionProperty<String, String> HOSTNAME_IN_CERTIFICATE = new HostnameInCertificate();
+    public static final ConnectionProperty<String, String> TIMEZONE = new TimeZone();
```

Comment:
Why not 
```suggestion
    public static final ConnectionProperty<String, ZoneId> TIMEZONE = new TimeZone();
```

Reply:
Awesome, this solved most of my concerns with the existing code. Thanks a lot.